java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/Constants.java

```diff
@@ -27,5 +27,6 @@ private Constants() {}
   public static final String SQL_TABLE_NAME_KEY = "SQL_TABLE_NAME";
   public static final String SQL_COLUMN_NAME_KEY = "SQL_COLUMN_NAME";
   public static final String SQL_TYPE_KEY = "SQL_TYPE";
+  public static final String COMMENT = "comment";
```

Comment:
Also, since the JDBC name for this is REMARKS, should we stay consistent with that?

Reply:
I prefer "comment" as it much more easy to don't introduce new names for de facto interoperability with Apache Spark https://github.com/apache/spark/blob/e83aedd0f072ce18d5542b61eb40e2df1b83bb50/sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructField.scala#L85 
Roundtrip conversion with org.apache.spark.sql.types.StructType/StructField and huge existing ecosystem around (AWS Glue etc)

Reply:
>>Also, since the JDBC name for this is REMARKS, should we stay consistent with that?
No, I not think so as it internal JDBC naming, but Arrow target following Spark/Hadoop but not JDBC

Reply:
Ah, if this is for spark compatibility that's good to know.

That said - I think my comment below makes more sense then, what do you think? We should just let you add whatever metadata instead of hardcoding particular fields